.gitignore

```diff
@@ -0,0 +1,3 @@
+
+/bazel-*
+*~
```

Comment:
As an emacs user, I appreciate this, but I prefer to put it in my own `~/.gitignore` and include that via `~/.gitconfig`:

```
[core]
        excludesfile = /expanded/path/to/~/.gitignore
```

That way I don't have to make such changes to every project I work on.

Reply:
makes sense - but i still think its a good idea for projects to carry this also